src/frontend/Komponenter/Behandling/Inngangsvilkår/InngangsvilkårHeader/InngangsvilkårHeader.tsx

```diff
@@ -1,101 +0,0 @@
-import React, { useCallback, useEffect, useState } from 'react';
-import styled from 'styled-components';
-import { OppdaterOpplysninger } from './OppdaterOpplysninger';
```

Comment:
Siden vi har fjernet denne, hvor har det blitt av bruken av `OppdaterOpplysninger` nå?

Reply:
Det samme gjelder `ÅpneOgLukkePanelKnapper`

Reply:
Bra sett! Det gikk litt for fort i svingene, dette skal være med.